deploy/vagrant/ufs/hadoop2/config_tachyon.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+namenode=$(tail -n1 /tachyon/conf/workers)
```

Comment:
Why is the formatting different from the command in the hadoop1 script? Also it would be good to not rely on confs outside of the deploy directory, but that can be dealt with in a separate PR.